### DISCUSSION TITLE
[Feature Request] HIGH SCORES

### DISCUSSION THREAD

**Discussion by david-p-s** [TeamRizu/OutFox#233] [03 - Feature Requests]
**Is your feature request related to a problem? Please describe.**

I can't find a way to show players' HighScores between sessions. When I exit the program and run it again, the previous statistics are not displayed. I would like to be able to know and see my record at all times, and not just by session.

**Describe the solution you'd like**

Show ./save/LocalProfiles/00000xxxx/Stats.xml parsed, similar to how this web shows http://dev.kserver.dy.fi/smpv/#

And also show the three best results of each song somewhere on the game screen to know what score I should beat at my next dance. 

**Describe alternatives you've considered**

---

**Additional context**
![2021-02-12_125141](https://user-images.githubusercontent.com/68908215/107775153-3b96b880-6d40-11eb-8b5d-c62008350604.jpg)

**Comment by ListenerJubatus** [comment]
This is something we will be doing in the upcoming new default theme.
